lxc/config/cert.go

```diff
@@ -1,7 +1,10 @@
 package config
 
 import (
+	"fmt"
 	"github.com/lxc/lxd/shared"
```

Comment:
We usually separate the builtin imports and the external ones, so this should become:
```
fmt
io
os

github.com/lxc/lxd/shared
```